### ISSUE TITLE
Implement ability to open/close DB connections not tied to web requests

### ISSUE THREAD

**Issue by ipolevoy** [javalite/activeweb#396]
Often in the webapps, there are other threads that do offline processing, from schedulers, Async, etc. 
They need to open/close the same connections as provided by https://github.com/javalite/activeweb/blob/master/activeweb/src/main/java/org/javalite/activeweb/controller_filters/DBConnectionFilter.java but not associated with a web request. 

Suggestion: implement a base class for  `DBConnectionFilter` and move discovery of configuration there.

**Comment by jacktang** [comment]
+1, and any progress on this?

**Comment by ipolevoy** [comment]
no, there is no progress on this yet, because Async does provide automatic database connection management: https://github.com/javalite/activeweb/blob/master/javalite-async/src/main/java/org/javalite/async/DBCommandListener.java and currently,  you can use the http://javalite.io/database_connection_management#use-try-with-resources which is pretty lightweight. Nonetheless, adding a class to manage connections still makes sense.
